scripts/e2e-ci.sh

```diff
@@ -362,7 +394,10 @@ test_exec() {
     wait_for_server shutdown $PID
 
     # evaluate tests
-    test_evaluate
+    # note we do not run eval on the small runner due to a lack of GPU resources
+    if [ "$SMALL" -eq 0 ]; then
```

Comment:
Have you tried to run evaluation on a small runner? Running MT-Bench should be possible with .gguf models, and if you have enough VRAM you could also run MMLU as well.

Reply:
MMLU didn't work for me with Granite safetensors (not enough VRAM) but I can def try MT Bench with GGUFs and see if we can add it as a followup